src/modlistsortproxy.h

```diff
@@ -72,7 +73,7 @@ class ModListSortProxy : public QSortFilterProxyModel
 
 public:
 
-  explicit ModListSortProxy(Profile *profile, QObject *parent = 0);
+  explicit ModListSortProxy(Profile *profile, OrganizerCore * organizer);

```

Comment:
I don't care which side you pick, but you gotta pick one.

Reply:
I don't understand your comment...?

Reply:
You have a space both before and after the asterisk.

Reply:
The age old debate over which side the asterisk belongs

Reply:
INTRUDER! SEIZE THEM!

Reply:
There is only one right answer to this: `T1* ptr, T2& ref`.

Reply:
Right, so I'm going to use `OrganizerCore *organizer`.